### PR TITLE
make jobs on linux64 debug displayable

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

File: src/treecodes.py [hunk 1]
```diff
@@ -213,15 +221,15 @@ def getGroup(name):
 
 def getGroupCode(name):
     try:
-        code = tbplnames[name]['group']
+        code = tbplnames[name.strip()]['group']
```
Comment (MikeLing) [inline src/treecodes.py]: some tests' name have useless blank (e.g ' mochitest-devtools-chrome-e10s-4') and it make we can't get the group symbol.

Comment (MikeLing) [inline src/treecodes.py]: please tell me if need a docstring for this :)

**Comment by MikeLing** [comment]
Maybe we could do this better after runnable api been landed.

File: src/treecodes.py [hunk 2]
```diff
@@ -42,11 +43,18 @@
...
              'mochitest-e10s-devtools-chrome-2': {'group': 'M-e10s', 'code': 'dt2'},
              'mochitest-e10s-devtools-chrome-3': {'group': 'M-e10s', 'code': 'dt3'},
              'mochitest-e10s-devtools-chrome-4': {'group': 'M-e10s', 'code': 'dt4'},
+             'mochitest-devtools-chrome-e10s-1': {'group': 'M-e10s', 'code': 'dt4'},
+             'mochitest-devtools-chrome-e10s-2': {'group': 'M-e10s', 'code': 'dt4'},
+             'mochitest-devtools-chrome-e10s-3': {'group': 'M-e10s', 'code': 'dt4'},
+             'mochitest-devtools-chrome-e10s-4': {'group': 'M-e10s', 'code': 'dt4'},
              'mochitest-push': {'group': 'M', 'code': 'p'},
              'xpcshell': {'group': '', 'code': 'X'},
              'xpcshell-1': {'group': '', 'code': 'X1'},
```
Comment (armenzg) [inline src/treecodes.py]: As I mentioned few times, I believe this data structure can be generated automatically.
It is pretty standard string manipulation.

File: src/treecodes.py (hunk 1 above)
Comment (armenzg) [inline src/treecodes.py]: Please add a docstring to the function.

**Pull request merged**